src/Turbo/Resources/config/doctrine_orm.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Kévin Dunglas <kevin@dunglas.fr>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\UX\Turbo\Broadcaster\BroadcasterInterface;
+use Symfony\UX\Turbo\Doctrine\BroadcastListener;
+
+/*
+ * @author Kévin Dunglas <kevin@dunglas.fr>
+ */
+return static function (ContainerConfigurator $container): void {
+    $container
```

Comment:
While I do like PHP config, I'm not sure why it's better than configuring it in the extension here, given the small size of concerned files. That would decrease the number of files to load.

WDYT?

Reply:
I've no strong opinion about this. WDYT @nicolas-grekas?

Reply:
I agree, we can remove the file. Or we might prefer doing the reverse as we do in Symfony:
have one config file and `removeDefinition` in the extension. This helps discoverability. (would apply to the other config files of course)

Reply:
@nicolas-grekas it's already done as described in your second solution. So I keep it as is?